src/secretsManager.ts

```diff
@@ -0,0 +1,36 @@
+import AWS from 'aws-sdk';
+
+const { IS_OFFLINE, AWS_REGION, STAGE } = process.env;
+export default class SecretsManager {
```

Comment:
Good question. I named it `SecretsManager` to describe it's job instead of it's implementation. This class is responsible for fetching secrets. The implementation detail is that it uses  `AWS Systems Manager`.

Reply:
Still, that name is very misleading. Also, what secrets are we fetching? The integration transform url is not a secret

Reply:
Updated to `ParameterStore`.